### PR TITLE
Article on smaller debug info data

### PR DESCRIPTION
**Pull request by noahp**

### CODE REVIEW

File: _drafts/dealing-with-large-symbol-files.md
Comment (noahp) [inline _drafts/dealing-with-large-symbol-files.md:455]: Goofed up the upx heading, need to fix

File: _drafts/dealing-with-large-symbol-files.md
Comment (franc0is) [inline _drafts/dealing-with-large-symbol-files.md:374]: I found this a little bit confusing: it reads as if `dwz` saved 50MB and `--compress-debug-sections` saved 91MB (so the latter is way better), but it also says that `dwz` saved 72% of the original, and `compress-debug-sections` saved 49% (so the latter is worse?). I think what you mean is that the resulting binary is 72%/49% of the original, but it's not super clear.

File: _drafts/dealing-with-large-symbol-files.md
Comment (franc0is) [reply _drafts/dealing-with-large-symbol-files.md:455]: This feels like a section/transition is missing?

**Review by franc0is — APPROVED** [review]
Another great article. Thanks @noahp !

File: _drafts/dealing-with-large-symbol-files.md
Comment (noahp) [reply _drafts/dealing-with-large-symbol-files.md:374]: Good call, this section is pretty awkward, might make better sense as a table

File: _drafts/dealing-with-large-symbol-files.md
Comment (noahp) [inline _drafts/dealing-with-large-symbol-files.md:159]: I biffed this example, need to redo, it's wrong.

File: _drafts/dealing-with-large-symbol-files.md
Comment (noahp) [inline _drafts/dealing-with-large-symbol-files.md:201]: _for_ example

File: _drafts/dealing-with-large-symbol-files.md
Comment (noahp) [inline _drafts/dealing-with-large-symbol-files.md:20]: add a note referencing https://interrupt.memfault.com/blog/gnu-binutils since we use a couple of those tools in this article

File: _drafts/dealing-with-large-symbol-files.md
Comment (tyhoff) [inline _drafts/dealing-with-large-symbol-files.md:478]: slowww -> maybe bold or italicize it instead?

File: _drafts/dealing-with-large-symbol-files.md
Comment (tyhoff) [inline _drafts/dealing-with-large-symbol-files.md:510]: Capitalize Impressive

**Review by tyhoff — APPROVED** [review]
Love the article!

**Comment by franc0is** [comment]
@noahp do you think we'll publish this tomorrow?

**Comment by noahp** [comment]
> 

Apologies, missed this! we can publish this, it's ready to go- maybe we should wait until compressed debug support is landed?

**Comment by chrisc11** [comment]
> > @noahp do you think we'll publish this tomorrow?
> 
> Apologies, missed this! we can publish this, it's ready to go- maybe we should wait until compressed debug support is landed?

yes, let's hold til that's live

**Pull request merged**
